test/e2e/run.sh

```diff
@@ -406,16 +429,27 @@ launch() { # script API
     #                Environment variables:
     #                cri_resmgr_cfg: configuration filepath (on host)
     #
+    #   cri-resmgr-agent:
+    #                launch cri-resmgr-agent on VM. Environment variables:
+    #                cri_resmgr_agent_extra_args: extra arguments on command line
+    #
+    #   cri-resmgr-webhook:
+    #                deploy cri-resmgr-webhook from the image on VM.
+    #
     # Example:
     #   cri_resmgr_cfg=/tmp/memtier.cfg launch cri-resmgr
-    target="$1"
+    local target="$1"
+    local launch_cmd
+    local adjustment_schema="$HOST_PROJECT_DIR/pkg/apis/resmgr/v1alpha1/adjustment-schema.yaml"
     case $target in
         "cri-resmgr")
             host-command "$SCP \"$cri_resmgr_cfg\" $VM_SSH_USER@$VM_IP:" || {
                 command-error "copying \"$cri_resmgr_cfg\" to VM failed"
             }
             vm-command "cat $(basename "$cri_resmgr_cfg")"
-            vm-command "cri-resmgr -relay-socket /var/run/cri-resmgr/cri-resmgr.sock -runtime-socket /var/run/containerd/containerd.sock -force-config $(basename "$cri_resmgr_cfg") $cri_resmgr_extra_args >cri-resmgr.output.txt 2>&1 &"
+            launch_cmd="cri-resmgr -relay-socket /var/run/cri-resmgr/cri-resmgr.sock -runtime-socket /var/run/containerd/containerd.sock -force-config $(basename "$cri_resmgr_cfg") $cri_resmgr_extra_args"
+            vm-command-q "echo '$launch_cmd' > cri-resmgr.launch.sh ; rm -f cri-resmgr.output.txt"
```

Comment:
What is the launch.sh for?

Reply:
Launch has been useful in manual testing & debugging: ssh to a VM and launch cri-resmgr just like it was previously launched by a test run.

Reply:
ok